code/__HELPERS/roundend.dm

```diff
@@ -366,8 +366,8 @@
  *
  * Composits the roundend report, and saves it in two locations.
  * The report is first saved along with the round's logs
- * Then, the report is copied to a fixed directory specifically for 
```

Comment:
lol why edit tg comments of all things

Reply:
I'll be honest, I don't know what's up, I didn't touch those comments. I might have to webedit as it's something I believe VSC did on its own and I'm unable to fix it through there.